SCClassLibrary/Common/Core/Error.sc

```diff
@@ -192,12 +192,12 @@ DoesNotUnderstandError : MethodError {
 }
 
 DoesNotUnderstandWithKeysError : MethodError {
-	var <>selector, <>argsWithOutKeys, <>argsWithKeys, <suggestedCorrection, suggestion = "";
-	*new { arg receiver, selector, argsWithOutKeys, argsWithKeys;
+	var <>selector, <>argumentsArray, <>keywordArgumentPairs, <suggestedCorrection, suggestion = "";
+	*new { arg receiver, selector, argumentsArray, keywordArgumentPairs;
 		^super.new(nil, receiver)
```

Comment:
if you want, you could write: `^super.newCopyArgs(selector, receiver, argumentsArray, keywordArgumentPairs).init`